org-roam-capture.el

```diff
@@ -279,23 +280,21 @@ This function is used solely in Org-roam's capture templates: see
 
 (defun org-roam-capture--convert-template (template)
   "Convert TEMPLATE from Org-roam syntax to `org-capture-templates' syntax."
-  (let* ((copy (copy-tree template))
-         converted
-         org-roam-plist
-         key
-         val)
-    ;;put positional args on converted template
-    (dotimes (_ 5)
-      (push (pop copy) converted))
-    (while (setq key (pop copy)
-                 val (pop copy))
-      (if (member key org-roam-capture--template-keywords)
-          (progn
-            (push val org-roam-plist)
-            (push key org-roam-plist))
-        (push key converted)
-        (push val converted)))
-    (append (nreverse converted) `(:org-roam ,org-roam-plist))))
+  (pcase template
+    (`(,key ,description) template)
```

Comment:
```suggestion
    (`(,_key ,_description) template)
```
silence byte-compiler warning about unused lexical args

Reply:
Same thing with the line below, I suppose.

Reply:
We're using the variables in the second case, so that should be fine.